RFS/src/testFixtures/java/org/opensearch/migrations/bulkload/framework/SearchClusterContainer.java

```diff
@@ -30,7 +30,7 @@ public class SearchClusterContainer extends GenericContainer<SearchClusterContai
         Version.fromString("ES 7.17.22")
     );
     public static final ContainerVersion ES_V6_8_23 = new ElasticsearchVersion(
-        "docker.elastic.co/elasticsearch/elasticsearch:6.8.23",
+        "docker.elastic.co/elasticsearch/elasticsearch-oss:6.8.23",
```

Comment:
Could you move all the ES versions to OSS and create a jira for adding tests for non-OSS ES versions in our OE epic?

Reply:
This does that as possible, there is no oss version post 7.10
https://opensearch.atlassian.net/browse/MIGRATIONS-2203